test/jdk/java/util/ResourceBundle/Control/MissingResourceCauseTestRun.java

```diff
@@ -50,9 +51,14 @@
 import jdk.test.lib.JDKToolLauncher;
 import jdk.test.lib.Utils;
 import jdk.test.lib.process.ProcessTools;
+import jdk.test.lib.Platform;
+import jtreg.SkippedException;
 
 public class MissingResourceCauseTestRun {
     public static void main(String[] args) throws Throwable {
+        if(Platform.isRoot() && !Platform.isWindows()) {
+            throw new SkippedException("root user has privileged will make this test fail.");
```

Comment:
The exception message can be improved. How about "Unable to create an unreadable properties file"?

Reply:
Thanks for the suggestion. The exception message has been change.